src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -541,6 +545,24 @@ pub struct Args {
     // === Tracing options. ===
     #[clap(flatten)]
     tracing: TracingCliArgs,
+
+    // === Testing options. ===
+    /// Injects arbitrary whitespace into builtin table fingerprints, which can
+    /// trigger builtin item migrations. The amount of whitespace is determined
+    /// by
+    /// `really_dangerous_do_not_call_this_in_production_table_fingerprint_whitespace_version`.
+    /// This argument is meant for testing only and as the names suggests
+    /// should not be set in production.
+    #[clap(long, arg_enum, requires = "unsafe-mode")]
+    really_dangerous_do_not_call_this_in_production_table_fingerprint_whitespace:
```

Comment:
Can we just call this `unsafe`, for symmetry with the existing `unsafe-mode` option? As fun as really-dangerous-dont-call-this-in-production is I think it's nice to have a single consistent term we use everywhere.

```suggestion
    unsafe_table_fingerprint_whitespace:
```

Reply:
Done